jupyter-notebooks/data-api-tutorials/search_windowed_read_and_clip_output.ipynb

```diff
@@ -7,16 +7,18 @@
    "source": [
     "# Accessing and displaying a sub-region of interest\n",
     "\n",
-    "Working with full Planet assets can be slow and cumbersome, which can create unnecessary overhead of you have a region of interest that is much smaller than the item footprint. Fortunately you can leverage [Cloud Optimized Geotiff](https://developers.planet.com/docs/planetschool/an-introduction-to-cloud-optimized-geotiffs-cogs-part-1-overview/) windowed reads to improve the speed of your Data API Asset Access workflow. Save storage and time by accessing only the pixels that you want to work with!\n",
+    "Working with full Planet assets can be slow and cumbersome if your region of interest is much smaller than an entire scene footprint. Fortunately, you can leverage [Cloud Optimized Geotiff](https://developers.planet.com/docs/planetschool/an-introduction-to-cloud-optimized-geotiffs-cogs-part-1-overview/) windowed reads to improve the speed of your Data API Asset Access workflow. Working with smaller regions of Planet assets can help you save storage and time.",
     "\n",
-    "This notebook shows how to work with smaller regions of Planet assets available via the Data API. In this tutorial we will demonstrate how to:\n",
+    "Please note that while this notebook's workflow is fast, it involves downloading full Planet scenes <b>BEFORE</b> clipping them to your Area of Interest; your account quota will be charged for the full size of each scene accessed, not just the final clipped portion. This is not as cost-effective as other methods.  If you are working with a large amount of data, do not use this notebook and instead consider the [Orders API Clipping Tool](https://developers.planet.com/apis/orders/tools/#clip).\n",
```

Comment:
The full scene should not actually downloaded, but some customers have quota plans that will charge for the entire scene when they read ANY data from the scene download link (even just the geotiff headers, for example). The problem is the pricing model for the data api usage is not COG-aware even though the data is COG-readable.

Here's some alternative wording
"Please note that while this notebook's workflow is fast, it involves accessing a download link for full Planet scenes. Accessing this link will charge you for downloading the entire scene even if you only actually download metadata or a small clipped portion of the asset. Due to the quota model in place for the Data API this is not as cost-effective as other methods. If you are working with a large amount of data and are concerned about quota, do not use this notebook and instead consider the [Orders API Clipping Tool](https://developers.planet.com/apis/orders/tools/#clip).\n"


See more on rasterio.open https://rasterio.readthedocs.io/en/latest/topics/reading.html
which relies on vsi https://rasterio.readthedocs.io/en/latest/topics/vsi.html
and more on COG https://www.cogeo.org/
> enable fully online processing of data by COG-aware clients, as they can stream the right parts of the GeoTIFF as they need it, instead of having to download the whole file.

Reply:
Thanks for the wording -- changed ✅